src/components/widgets/QRCode.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import "../../styles/widget/widget.css"
+import qrCode from 'qrcode-npm'
+import Header from './Header'
+
+const QRCodeImage = ( url ) => {
```

Comment:
I vaguely recall discussing this, but not sure what advice I gave last.  :)  I'm *currently* thinking this might be more appropriate:
```jsx
export default class QRCodeImage extends Component {
  static defaultProps = {
    width: 164,
    height: 164
  }

  qrSource() {
    const qr = QRCodeGenerator.qrCode( 4, 'M' )
 
    qr.addData( this.props.url )
    qr.make()
 
    return imageTag.replace( '" width="164" height="164"/>', '' ).replace( '<img src="', '' )
  }

  render() {
    return <img src={this.qrSource()} 
      width={this.props.width} 
      height={this.props.height} 
      alt="QR Code" />
  }
}
```

Now we can have the `QRCode` component just render this without having additional functionality in the module:
```js
  <QRCodeImage url={this.props.url} />
```